modules/nw-dual-stack-convert.adoc

```diff
@@ -33,36 +34,59 @@ After converting to dual-stack networking only newly created pods are assigned I
   path: /spec/serviceNetwork/-
   value: fd02::/112 <2>
 ----
-<1> Specify an object with the `cidr` and `hostPrefix` fields. The host prefix must be `64` or greater. The IPv6 CIDR prefix must be large enough to accommodate the specified host prefix.
-
+<1> Specify an object with the `cidr` and `hostPrefix` fields. The host prefix must be `64` or greater. The IPv6 Classless Inter-Domain Routing (CIDR) prefix must be large enough to accommodate the specified host prefix.
 <2> Specify an IPv6 CIDR with a prefix of `112`. Kubernetes uses only the lowest 16 bits. For a prefix of `112`, IP addresses are assigned from `112` to `128` bits.
---
 
-. To patch the cluster network configuration, enter the following command:
+. To patch the cluster network configuration, enter the following command in your CLI:
```

Comment:
Just suggesting the most common way to do this. In any case, don't need "in your CLI".
```suggestion
. Patch the cluster network configuration by running the following command:
```

Reply:
Thanks. I went with "entering": 

![Screenshot from 2024-05-09 08-57-05](https://github.com/openshift/openshift-docs/assets/57954076/86471ca4-efb7-4216-b2e5-e6bca3b72b22)

Reply:
Yep, either is fine. I tend to use "running" since it's by far the most common in the repo (for consistency), but "entering" is valid per our guidelines :)